README.md

```diff
@@ -5,6 +5,25 @@ By storing them in a central place (here), and distributing them in an automated
 1. Consistency: Every participating repository uses the same workflow, ensuring that our code adheres to the same coding standards and is properly tested.
 2. Maintainability: Workflows change over time. We need to be able to make changes without manually updating dozens of repositories.
 
+## Customization
+
+Most repositories won't need any customization, and the workflows defined here will just work fine.
+Some repositories may require some pre-setup steps to be run before tests (or code checks) can be run. Setup steps for `go-test` are defined in `.github/actions/go-test_setup/action.yml`, and setup steps for `go-check` are defined in `.github/actions/go-check_setup/action.yml`, in the following format:
+
+```yml
```

Comment:
Once this is used by any of our repos, we could link to that repo for a real-life example.